tests/vessels/test_generator.toit

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2022 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+import host.file
+
+DATA_256 := ((List 256: it).map: "$(%3d it), ").join ""
```

Comment:
Add a comment that shows the shape of the string, so every reader do not have to run this through their head.

Reply:
done.